examples/passport-login/README.md

```diff
@@ -200,3 +204,11 @@ http://loopback.io/doc/en/lb2/Configuring-providers.json.html.
       "failureFlash": true
   }
   ```
+
+### Log in with Google
+
+- Open your browser to the example app with, `http://localhost:3000`
+- Click on 'Log In' from the example app header menu
```

Comment:
I believe we usually use back-tick instead. 

```suggestion
- Click `Log In` from the example app header menu
```

Reply:
agreed